test/onnx/test_pytorch_onnx_onnxruntime_cuda.py

```diff
@@ -8,17 +8,14 @@
 import parameterized
 
 import torch
+from onnx_test_common import MAX_ONNX_OPSET_VERSION, MIN_ONNX_OPSET_VERSION
```

Comment:
why these changes?

Reply:
Those constants aren't defined in the file they were being imported from. This file must not be run in CI.

Reply:
mind filing an issue?

Reply:
Thanks!